frequently-asked-questions.md

```diff
@@ -30,10 +30,12 @@ client.on("ready", () => {
 // Set the bot's presence (activity and status)
 client.on("ready", () => {
     client.user.setPresence({
-        activities: [{ 
+        activities: [
```

Comment:
Why newline the array?
It just makes the code uglier...

Reply:
Thought it would look better. Never mind though, I'll just fix it.